litrpc/netcmds.go

```diff
@@ -64,6 +65,25 @@ func (r *LitRPC) Connect(args ConnectArgs, reply *StatusReply) error {
 	return nil
 }
 
+// ------------------------- name a connection
+type AssignNicknameArgs struct {
+	Peer     uint32
+	Nickname string
+}
+
+func (r *LitRPC) AssignNickname(args AssignNicknameArgs, reply *StatusReply) error {
+	err := r.Node.SaveNicknameForPeerIdx(args.Nickname, args.Peer)
```

Comment:
Can you add a comment on why you need to call SaveNicknameForPeerIdx and then also set Nickname on r.Node.RemoteCons?